gsplat/experimental/cuda/_torch_impl.py

```diff
@@ -0,0 +1,290 @@
+import struct
+from typing import Tuple
+
+import torch
+from torch import Tensor
+
+
+def _quat_scale_to_covar_perci(
```

Comment:
why `perci` over `preci` or `precis`?

Reply:
thanks for catching it yeah its a typo should be preci